bin/subst.ml

```diff
@@ -16,7 +16,18 @@ let is_a_source_file path =
    | ".ttf"
    | ".woff" -> false
    | _ -> true)
-  && (Path.stat_exn path).st_kind = Unix.S_REG
+
+let is_kind_a_source_file path=
+  match Path.stat_exn path with
```

Comment:
Maybe use `Path.stat` instead of `stat_exn`?

Reply:
I'm a bit torn about this one this we still need to reraise the other error cases. I pushed a commit, let me know what you think.

Reply:
Ah, that's a good point. That said, I think the current state looks good to me and I am in favor of merging it.